src/main/java/ai/datahunters/md/server/analytics/AnalyticsController.java

```diff
@@ -0,0 +1,24 @@
+package ai.datahunters.md.server.analytics;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+import java.util.concurrent.ExecutionException;
+
+@AllArgsConstructor
+@RestController
+public class AnalyticsController {
+
+    private AnalyticsHandler handler;
+
+    @GetMapping(value = "/api/v1/analytics", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I forgot, can you add a CORS header, the same as we have in photos endpoint `     @CrossOrigin(origins = "*") ` over this endpoint?